### ISSUE TITLE
Clippy Error

### ISSUE THREAD

**Issue by ski0090** [rust-lang/rust-clippy#11652]
### Summary

Unable to determine where the error occurred

### Version

```text
rustc 1.73.0 (cc66ad468 2023-10-03)
binary: rustc
commit-hash: cc66ad468955717ab92600c770da8c1601a4ff33
commit-date: 2023-10-03
host: x86_64-pc-windows-msvc
release: 1.73.0
LLVM version: 17.0.2
```

### Error output
`$env:RUST_BACKTRACE="full" & cargo clippy`
<details><summary>Backtrace </summary>
  <p>

  ```
thread 'rustc' panicked at src\tools\clippy\clippy_lints\src\redundant_locals.rs:67:49:
called `Option::unwrap()` on a `None` value
stack backtrace:
   0:     0x7fffd717738a - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h90c22c4185b55f04
   1:     0x7fffd71abb5b - core::fmt::write::h8a7f6edbcadee319
   2:     0x7fffd716cc51 - <std::io::IoSlice as core::fmt::Debug>::fmt::h0ee392ef0f96e369
   3:     0x7fffd717710a - std::sys_common::backtrace::lock::h134eeaadb35136bd
   4:     0x7fffd717a8ca - std::panicking::default_hook::h81d7caaa8c7a40b4
   5:     0x7fffd717a538 - std::panicking::default_hook::h81d7caaa8c7a40b4
   6:     0x7fffdba3d77e - <tracing_subscriber[610e65fa1e67acd7]::fmt::format::Writer>::write_fmt
   7:     0x7fffd717b1cc - std::panicking::rust_panic_with_hook::h8ac326f3b5470b08
   8:     0x7fffd717aeea - <std::panicking::begin_panic_handler::StrPanicPayload as core::panic::BoxMeUp>::get::h353ca2f9698f73b3
   9:     0x7fffd7178049 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h90c22c4185b55f04
  10:     0x7fffd717ac30 - rust_begin_unwind
  11:     0x7fffd71e0d85 - core::panicking::panic_fmt::hb3ce2a19a4187054
  12:     0x7fffd71e0e32 - core::panicking::panic::h5206d844b81664c1
  13:     0x7ff66cf88f14 - <unknown>
  14:     0x7fffdc88d908 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  15:     0x7fffdc8c320c - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  16:     0x7fffdc88d9b7 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  17:     0x7fffdc88d2af - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  18:     0x7fffdc88d9b7 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  19:     0x7fffdc88d2af - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  20:     0x7fffdc88d9b7 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  21:     0x7fffdc88d2af - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  22:     0x7fffdc88cfb4 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  23:     0x7fffdc88d4fd - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  24:     0x7fffdc8c5f48 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  25:     0x7fffdc88d2af - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  26:     0x7fffdc88d9b7 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  27:     0x7fffdc88d2af - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  28:     0x7fffdc8c5e36 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  29:     0x7fffdc88d2af - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  30:     0x7fffdc8c320c - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  31:     0x7fffdc88d9b7 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  32:     0x7fffdc88d2af - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  33:     0x7fffdc88cfb4 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  34:     0x7fffdc88d4fd - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  35:     0x7fffdc8c3918 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  36:     0x7fffdc88b9a3 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  37:     0x7fffdc8c6723 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  38:     0x7fffdc88b812 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  39:     0x7fffdc8c560b - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  40:     0x7fffdc8c6574 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  41:     0x7fffdc88b812 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  42:     0x7fffdc8c560b - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  43:     0x7fffdc8c6574 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  44:     0x7fffdc88b812 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  45:     0x7fffdc8c560b - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  46:     0x7fffdc8c6574 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  47:     0x7fffdc88b812 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  48:     0x7fffdc8c560b - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  49:     0x7fffdc8c6574 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  50:     0x7fffdc88b812 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  51:     0x7fffdc8c560b - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  52:     0x7fffdc8c6574 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  53:     0x7fffdc88b812 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  54:     0x7fffdc8c560b - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  55:     0x7fffdc880456 - <rustc_lint[582c04f3e3b29df3]::ptr_nulls::PtrNullChecks as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  56:     0x7fffda921676 - <rustc_lint[582c04f3e3b29df3]::levels::LintLevelsBuilder<rustc_lint[582c04f3e3b29df3]::levels::LintLevelQueryMap> as rustc_hir[28cfdbc768b2a7b2]::intravisit::Visitor>::visit_field_def
  57:     0x7fffda923d2d - rustc_lint[582c04f3e3b29df3]::late::check_crate
  58:     0x7fffd7c47ff1 - time_core[9f5980238a4d8d4f]::util::is_leap_year
  59:     0x7fffd7c48bb1 - time_core[9f5980238a4d8d4f]::util::is_leap_year
  60:     0x7fffd7c46253 - time_core[9f5980238a4d8d4f]::util::is_leap_year
  61:     0x7fffd7c62173 - rustc_interface[1fce9354a182ed1c]::passes::analysis
  62:     0x7fffda68ee7d - rustc_query_impl[4ba43568455dd7ab]::profiling_support::alloc_self_profile_query_strings
  63:     0x7fffda6a2849 - rustc_query_impl[4ba43568455dd7ab]::profiling_support::alloc_self_profile_query_strings
  64:     0x7fffda6a48b5 - rustc_query_impl[4ba43568455dd7ab]::profiling_support::alloc_self_profile_query_strings
  65:     0x7fffda6f8637 - rustc_query_impl[4ba43568455dd7ab]::profiling_support::alloc_self_profile_query_strings
  66:     0x7fffda6bdbac - rustc_query_impl[4ba43568455dd7ab]::profiling_support::alloc_self_profile_query_strings
  67:     0x7fffd7c0d891 - rustc_driver_impl[1e281a1232643afa]::args::arg_expand_all
  68:     0x7fffd7c1bd6d - <rustc_middle[b5332b31642963de]::ty::SymbolName as core[480aa84f3feba9e5]::fmt::Debug>::fmt
  69:     0x7fffd7c191c6 - <rustc_middle[b5332b31642963de]::ty::SymbolName as core[480aa84f3feba9e5]::fmt::Debug>::fmt
  70:     0x7fffd7c1a806 - <rustc_middle[b5332b31642963de]::ty::SymbolName as core[480aa84f3feba9e5]::fmt::Debug>::fmt
  71:     0x7fffd718e3ec - std::sys::windows::thread::Thread::new::h75a9ed52a5af2748
  72:     0x7ff89c1c257d - BaseThreadInitThunk
  73:     0x7ff89d64aa68 - RtlUserThreadStart

error: the compiler unexpectedly panicked. this is a bug.

note: we would appreciate a bug report: https://github.com/rust-lang/rust-clippy/issues/new?template=ice.yml

note: rustc 1.73.0 (cc66ad468 2023-10-03) running on x86_64-pc-windows-msvc

note: compiler flags: -C embed-bitcode=no -C debuginfo=2 -C incremental=[REDACTED]

note: some of the compiler flags provided by cargo are hidden

query stack during panic:
#0 [analysis] running analysis passes on this crate
end of query stack
note: Clippy version: clippy 0.1.73 (cc66ad46 2023-10-03)

error: could not compile `at-nvr` (bin "at-nvr" test)
warning: build failed, waiting for other jobs to finish...
thread 'rustc' panicked at src\tools\clippy\clippy_lints\src\redundant_locals.rs:67:49:
called `Option::unwrap()` on a `None` value
stack backtrace:
   0:     0x7fffd717738a - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h90c22c4185b55f04
   1:     0x7fffd71abb5b - core::fmt::write::h8a7f6edbcadee319
   2:     0x7fffd716cc51 - <std::io::IoSlice as core::fmt::Debug>::fmt::h0ee392ef0f96e369
   3:     0x7fffd717710a - std::sys_common::backtrace::lock::h134eeaadb35136bd
   4:     0x7fffd717a8ca - std::panicking::default_hook::h81d7caaa8c7a40b4
   5:     0x7fffd717a538 - std::panicking::default_hook::h81d7caaa8c7a40b4
   6:     0x7fffdba3d77e - <tracing_subscriber[610e65fa1e67acd7]::fmt::format::Writer>::write_fmt
   7:     0x7fffd717b1cc - std::panicking::rust_panic_with_hook::h8ac326f3b5470b08
   8:     0x7fffd717aeea - <std::panicking::begin_panic_handler::StrPanicPayload as core::panic::BoxMeUp>::get::h353ca2f9698f73b3
   9:     0x7fffd7178049 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h90c22c4185b55f04
  10:     0x7fffd717ac30 - rust_begin_unwind
  11:     0x7fffd71e0d85 - core::panicking::panic_fmt::hb3ce2a19a4187054
  12:     0x7fffd71e0e32 - core::panicking::panic::h5206d844b81664c1
  13:     0x7ff66cf88f14 - <unknown>
  14:     0x7fffdc88d908 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  15:     0x7fffdc8c320c - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  16:     0x7fffdc88d9b7 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  17:     0x7fffdc88d2af - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  18:     0x7fffdc88d9b7 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  19:     0x7fffdc88d2af - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  20:     0x7fffdc88d9b7 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  21:     0x7fffdc88d2af - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  22:     0x7fffdc88cfb4 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  23:     0x7fffdc88d4fd - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  24:     0x7fffdc8c5f48 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  25:     0x7fffdc88d2af - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  26:     0x7fffdc88d9b7 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  27:     0x7fffdc88d2af - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  28:     0x7fffdc8c5e36 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  29:     0x7fffdc88d2af - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  30:     0x7fffdc8c320c - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  31:     0x7fffdc88d9b7 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  32:     0x7fffdc88d2af - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  33:     0x7fffdc88cfb4 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  34:     0x7fffdc88d4fd - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  35:     0x7fffdc8c3918 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  36:     0x7fffdc88b9a3 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  37:     0x7fffdc8c6723 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  38:     0x7fffdc88b812 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  39:     0x7fffdc8c560b - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  40:     0x7fffdc8c6574 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  41:     0x7fffdc88b812 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  42:     0x7fffdc8c560b - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  43:     0x7fffdc8c6574 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  44:     0x7fffdc88b812 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  45:     0x7fffdc8c560b - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  46:     0x7fffdc8c6574 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  47:     0x7fffdc88b812 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  48:     0x7fffdc8c560b - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  49:     0x7fffdc8c6574 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  50:     0x7fffdc88b812 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  51:     0x7fffdc8c560b - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  52:     0x7fffdc8c6574 - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  53:     0x7fffdc88b812 - <rustc_lint[582c04f3e3b29df3]::non_ascii_idents::NonAsciiIdents as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  54:     0x7fffdc8c560b - <rustc_lint[582c04f3e3b29df3]::lints::UnusedAllocationMutDiag as rustc_errors[c0d41f87207b64dd]::diagnostic::DecorateLint<()>>::msg
  55:     0x7fffdc880456 - <rustc_lint[582c04f3e3b29df3]::ptr_nulls::PtrNullChecks as rustc_lint_defs[3237e8d38bc2e3e5]::LintPass>::name
  56:     0x7fffda921676 - <rustc_lint[582c04f3e3b29df3]::levels::LintLevelsBuilder<rustc_lint[582c04f3e3b29df3]::levels::LintLevelQueryMap> as rustc_hir[28cfdbc768b2a7b2]::intravisit::Visitor>::visit_field_def
  57:     0x7fffda923d2d - rustc_lint[582c04f3e3b29df3]::late::check_crate
  58:     0x7fffd7c47ff1 - time_core[9f5980238a4d8d4f]::util::is_leap_year
  59:     0x7fffd7c48bb1 - time_core[9f5980238a4d8d4f]::util::is_leap_year
  60:     0x7fffd7c46253 - time_core[9f5980238a4d8d4f]::util::is_leap_year
  61:     0x7fffd7c62173 - rustc_interface[1fce9354a182ed1c]::passes::analysis
  62:     0x7fffda68ee7d - rustc_query_impl[4ba43568455dd7ab]::profiling_support::alloc_self_profile_query_strings
  63:     0x7fffda6a2849 - rustc_query_impl[4ba43568455dd7ab]::profiling_support::alloc_self_profile_query_strings
  64:     0x7fffda6a48b5 - rustc_query_impl[4ba43568455dd7ab]::profiling_support::alloc_self_profile_query_strings
  65:     0x7fffda6f8637 - rustc_query_impl[4ba43568455dd7ab]::profiling_support::alloc_self_profile_query_strings
  66:     0x7fffda6bdbac - rustc_query_impl[4ba43568455dd7ab]::profiling_support::alloc_self_profile_query_strings
  67:     0x7fffd7c0d891 - rustc_driver_impl[1e281a1232643afa]::args::arg_expand_all
  68:     0x7fffd7c1bd6d - <rustc_middle[b5332b31642963de]::ty::SymbolName as core[480aa84f3feba9e5]::fmt::Debug>::fmt
  69:     0x7fffd7c191c6 - <rustc_middle[b5332b31642963de]::ty::SymbolName as core[480aa84f3feba9e5]::fmt::Debug>::fmt
  70:     0x7fffd7c1a806 - <rustc_middle[b5332b31642963de]::ty::SymbolName as core[480aa84f3feba9e5]::fmt::Debug>::fmt
  71:     0x7fffd718e3ec - std::sys::windows::thread::Thread::new::h75a9ed52a5af2748
  72:     0x7ff89c1c257d - BaseThreadInitThunk
  73:     0x7ff89d64aa68 - RtlUserThreadStart

error: the compiler unexpectedly panicked. this is a bug.

note: we would appreciate a bug report: https://github.com/rust-lang/rust-clippy/issues/new?template=ice.yml

note: rustc 1.73.0 (cc66ad468 2023-10-03) running on x86_64-pc-windows-msvc

note: compiler flags: --crate-type bin -C embed-bitcode=no -C debuginfo=2 -C incremental=[REDACTED]

note: some of the compiler flags provided by cargo are hidden

query stack during panic:
#0 [analysis] running analysis passes on this crate
end of query stack
note: Clippy version: clippy 0.1.73 (cc66ad46 2023-10-03)
  ```

  </p>
</details>

**Comment by y21** [comment]
Based on the panic message, this is probably a duplicate of #11619, which was already fixed in #11623 (the unwrap that failed here was removed)

**Comment by ski0090** [comment]
> 패닉 메시지에 따르면 이것은 아마도 #11619에서 이미 수정된 #11623의 중복일 것입니다 (여기서 실패한 래핑 해제는 제거됨)

thanks!
